### DISCUSSION TITLE
Error while trying to containerize

### DISCUSSION THREAD

**Discussion by oshinowo-stephen** [Pryaxis/TShock#2665] [General]
Originally presnted [here](https://github.com/ryansheehan/terraria/issues/114), and the [crash](https://github.com/oshinowo-stephen/world-mania/blob/terraria/terraria/FrenchFries.wld.crash) file. I'm still not certain why i'm running into such error nor is it presenting me with enough information to distinct the problem any pointers would be really helpful to resolve this...

**Comment by hakusaro** [comment]
Can you try this solution? It looks like you’ve hit a problem that might be trivially solved with a simple command-line argument change: https://github.com/Pryaxis/TShock/discussions/2524#discussioncomment-1726321
